agent/connect/ca/provider.go

```diff
@@ -193,7 +194,12 @@ type SecondaryProvider interface {
 //
 // TODO: rename this struct
 type RootResult struct {
-	// PEM encoded certificate that will be used as the primary CA.
+	// PEM encoded bundle of CA certificates. The first certificate must be the
+	// primary CA used to sign intermediates for secondary datacenters, and the
+	// last certificate must be the trusted CA.
```

Comment:
If this can return multiple certificates in a chain now, could that cause any problems for downstream things that call the `/ca/roots` endpoint?

Reply:
Along a similar line of reasoning, if someone doesn't switch to an external trusted CA immediately, would upgrading to a patched version of Consul ever _start_ returning multiple certs in a chain? If so that may break during the rolling upgrade of consul from a non-patched to a patched version.

Reply:
Right, it won't change in that case. The only way this starts returning multiple certs is when the user updates the config to do so. Upgrading Consul continues to return a single cert in this PEM.  

Existing versions of Consul also return multiple certs here if the `RootCert` is configured that way. I think that might already work with the built-in provider. That's why I think this is mostly just a docs change. It could have already been returning multiple certs, it just wasn't obvious before.